src/solidity/astHelper.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+var AstWalker = require('../util/astWalker')
+
+/**
+  * return all contract definitions of the given @astList
+  *
+  * @param {Object} astList  - AST nodes of all the sources
+  * @return {Object} - return the AST node of found contracts
```

Comment:
Please be a bit more specific about the types. If you want to know how to use the return value of that function, you have to read the source code. What about `@return {Object} returns a mapping from AST node ids to AST nodes for the contracts`.